tachyon/math/base/big_int.h

```diff
@@ -218,6 +230,7 @@ struct ALIGNAS(internal::LimbsAlignment(N)) BigInt {
     return FromMontgomery(value, modulus, inverse);
   }
 
+  // Extend the current N size BigInt to a larger N2 size.
```

Comment:
How about |N|, |N2| for consistency? If you agree, could you also make same changes to other comments as well?